pipeline.groovy

```diff
@@ -134,6 +135,20 @@ for(i in pzprojects) {
         defaultValue("nexus-deployment")
         description("Nexus Repository Credentials")
       }
+      if (i.requiresJksCreds) {
```

Comment:
I would expect null to evaluate to false, since this value isn't included for most of the projects